core/input/input_map.cpp

```diff
@@ -61,12 +61,11 @@ void InputMap::_bind_methods() {
  * matching action name (if possible).
  */
 String InputMap::suggest_actions(const StringName &p_action) const {
-	List<StringName> actions = get_actions();
 	StringName closest_action;
 	float closest_similarity = 0.0;
 
 	// Find the most action with the most similar name.
-	for (const StringName &action : actions) {
+	for (const Variant &action : get_actions()) {
```

Comment:
Don't we just want to iterate `input_map`? I feel like copying to an array is a waste of time here.